### ISSUE TITLE
[awesomehd] Found no results while trying to browse this tracker (Test)

### ISSUE THREAD

**Issue by rodepanda** [Jackett/Jackett#7861]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed.

Please read our [Contributing Guidelines](https://github.com/Jackett/Jackett/blob/master/CONTRIBUTING.md) before submitting your issue to ensure a prompt response to your bug.

### Environment

**OS**: Debian Buster

**.Net Runtime/Version**: Not sure I use a dockercontainer from here: https://hub.docker.com/r/linuxserver/jackett

**Jackett Version**: 0.14.287.0

**Last Working Jackett Version**: Not sure

**Are you using a proxy or VPN?** no

### Description

I click on the test button from awesomehd after which it errors

### Logged Error Messages
```
2020-03-26 19:40:57 | Error | System.Exception: Found no results while trying to browse this tracker    at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in /home/vsts/work/1/s/src/Jackett.Common/Services/IndexerManagerService.cs:line 224    at Jackett.Server.Controllers.IndexerApiController.Test() in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 130    at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited\|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited\|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited\|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited\|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited\|17_0(ResourceInvoker invoker, Task task, IDisposable scope)    at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask\|6_0(Endpoint endpoint, Task requestTask, ILogger logger)    at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)    at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 70
-- | -- | --
2020-03-26 19:40:57 | Info | Found 0 releases from Awesome-HD
2020-03-26 19:40:57 | Error | CardigannIndexer (awesomehd): Error while parsing row ' <tr class="group_torrent groupid_41224"> 	<td colspan="3"> 		<span> 			[<a href="torrents.php?action=download&amp;id=90331&amp;authkey=&&&&amp;torrent_pass=&&&" title="Download">DL</a> 			\| 			<a href="reportsv2.php?action=report&amp;id=90331" title="Report">RP</a>] 		</span> 		» 		<a href="torrents.php?id=41224&amp;torrentid=90331"> 			<strong>NTG</strong> 			/ h.264 Remux / 1080p / WEB-DL / E-AC-3 / Amazon / 			<strong>DXVA</strong> 		</a> 	</td> 	<td>1</td> 	<td class="nobr"> 		<span title="Mar 25 2020, 00:13">1 day ago</span> 	</td> 	<td class="nobr">3.13 GB</td> 	<td>4</td> 	<td>5</td> 	<td>0</td> </tr>':  System.Exception: Error while parsing field=title, selector=td:nth-child(3) > a, value=<null>: Selector "td:nth-child(3) > a" didn't match  <tr class="group_torrent groupid_41224"> 	<td colspan="3"> 		<span> 			[<a href="torrents.php?action=download&amp;id=90331&amp;authkey=&&&&&amp;torrent_pass=&&&&" title="Download">DL</a> 			\| 			<a href="reportsv2.php?action=report&amp;id=90331" title="Report">RP</a>] 		</span> 		» 		<a href="torrents.php?id=41224&amp;torrentid=90331"> 			<strong>NTG</strong> 			/ h.264 Remux / 1080p / WEB-DL / E-AC-3 / Amazon / 			<strong>DXVA</strong> 		</a> 	</td> 	<td>1</td> 	<td class="nobr"> 		<span title="Mar 25 2020, 00:13">1 day ago</span> 	</td> 	<td class="nobr">3.13 GB</td> 	<td>4</td> 	<td>5</td> 	<td>0</td> </tr>    at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1575
2020-03-26 19:40:57 | Error | CardigannIndexer (awesomehd): Error while parsing row ' <tr class="group_torrent groupid_41224"> 	<td colspan="9" class="edition_info"> 		<strong> 			- Amazon</strong> 	</td> </tr>':  System.Exception: Error while parsing field=details, selector=a[href^="torrents.php?id="], value=<null>: Selector "a[href^="torrents.php?id="]" didn't match  <tr class="group_torrent groupid_41224"> 	<td colspan="9" class="edition_info"> 		<strong> 			- Amazon</strong> 	</td> </tr>    at Jackett.Common.Indexers.CardigannIndexer.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1575
```
### Screenshots

[Place any screenshots of the issue here if needed]

**Comment by RoloSoze** [comment]
#7829 #7827

**Comment by garfield69** [comment]
resolved by #7827
